package.json

```diff
@@ -62,6 +63,7 @@
     "node": ">=10.10.0"
   },
   "dependencies": {
+    "@azure/cognitiveservices-luis-authoring": "^4.0.0-preview.1",
```

Comment:
This package.json file corresponds to the entire mono repo.
You shouldn't need to make any edits here.
Any dependencies you need should be added to the package.json file in the `cognitiveservices-luis-authoring` folder

Reply:
thanks for the clarification,
changes reverted